src/System.CodeDom/src/System/CodeDom/Compiler/CodeDomProvider.cs

```diff
@@ -11,7 +11,7 @@
 
 namespace System.CodeDom.Compiler
 {
-    public abstract class CodeDomProvider // TODO: Inherit Component
+    public abstract class CodeDomProvider: Component
```

Comment:
Nit: please add a space before the colon

This base class should also be added to the ref at:
https://github.com/marek-safar/corefx/blob/4928ffe675b4a4b3c0bb7f0d29aacbfc950b95f1/src/System.CodeDom/ref/System.CodeDom.cs#L879